### ISSUE TITLE
Disconnected from the server."

### ISSUE THREAD

**Issue by holgerbrandl** [databio/LOLAweb#95]
![image](https://user-images.githubusercontent.com/200952/46296435-a9d8e700-c59a-11e8-9a63-c318ecc815be.png)

I don't know the precise timing, but it seems to die before starting "Calculating universe set overlaps..."

**Comment by holgerbrandl** [comment]
Duplicate of #96
